---
layout: page
title: Research
---

<p align="justify"> It has long been known that mutant clonal evolution by natural selection plays a defining role in the development of cancer. However, this process has been challenging to study in detail prior to the development of cost-effective high-throughput Next-Generation Sequencing. Using NGS, alterations present in even very small proportions of cells can now be detected. Modelling these alterations can in turn improve our understanding of the emergence of mutant alleles within tissues. Moreover, recently developed single cell RNA sequencing methods can be used to identify somatic mutations in individual cells and assess their effects on the expression of genes, an approach that has been highly informative in the interpretation of the effects of germline variation. A major component of our work is therefore to interpret and develop novel methods for the analysis of NGS data in the context of cancer evolution.

Another component of our work is the more general understanding and prediction of genotype to phenotype relationships. For this, we contribute through the development of computational methods for high-dimensional data, including tools for feature selection and differential testing. </p>

### Projects

Some of our projects:
* Dynamics of somatic mutation in normal and pre-malignant human gynaecological tissues
* Comparative genomics of female reproductive tissue ageing
* Methods development for single-cell RNA-seq: 
  * KIRid/KIRquant
  * single-cell differential distributions
  * lineage tracing
* Effects of common genetic variants on iPSC epigenomes 



