---
layout: page
title: Publications
---

### Methods/software

* Schefzik R., Flesch J., Goncalves A. (2020). [Fast identification of differential distributions in single-cell RNA-sequencing data with waddR](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btab226/6207964). Bioinformatics

* Vento-Tormo R., Efremova M., Turco M.Y., Botting R.A., Meyer K.B., Park J., Stephenson E., Payne R.P., Goncalves A., Zou A., Henriksson J., Wood L., Lisgo S., Filby A., Wright G.J., Stubbington M.J.T., Haniffa M., Moffett A., Teichmann S.A. (2018). [Reconstructing the human first trimester fetal-maternal interface using single cell transcriptomics](https://doi.org/10.1038/s41586-018-0698-6). Nature. 546, 370-375.

*	Turro E., Su S., Goncalves A., Coin L.J.M, Richardson S. and Lewin A. (2011) [Haplotype and isoform specific expression estimation using multi-mapping RNA-seq reads](https://doi.org/10.1186/gb-2011-12-2-r13). Genome Biology 12 (2), 1. 

*	Goncalves A., Tikhonov A., Brazma A. and Kapushesky M. (2011) [A pipeline for RNA-seq data processing and quality assessment](https://doi.org/10.1093/bioinformatics/btr012). Bioinformatics 27 (6), 867-869. 

### Population genomics

* Schwartzentruber J., Foskolou S., Kilpinen H., Rodrigues J., Alasoo K., Knights A.J., Patel M., Goncalves A., Ferreira R., Benn C.L., Wilbrey A., Bictash M., Impey E., Cao L., Lainez S., Loucif AJ., Whiting P.J., HIPSCI Consortium, Gutteridge A. and Gaffney D. (2017) [Molecular and functional variation in iPSC-derived sensory neurons](https://doi.org/10.1038/s41588-017-0005-8
). Nature Genetics. 50, 54-61. 

* Kilpinen H.#, Goncalves A.#, Leha A., Afzal V., Bensaddek D., Casale F.P., Danecek P., Harrison P., McCarthy D., McCarthy S., Meleckyte R., Memari Y., Moens N., Soares F., Streeter I., Agu C., Alderton A., Amatya S., Nelson R., Harper S., Patel M., Clarke L., Halai R., Kirton C., Kolb-Kokocinski A., Beales P., Birney E., Danovi D., Lamond A., Ouwehand W., Vallier L., Watt F., Durbin R., Stegle O. and Gaffney D. (2017) [Common genetic variation drives molecular heterogeneity in human IPSCs](https://doi.org/10.1038/nature22403). Nature 546 (7658), 370. 

### Gene expression regulation 

*	Leigh-Brown S.#, Goncalves A.#, Thybert D., Stefflova K., Watt S., Odom D.T. and Marioni J.C. (2015) [Regulatory Divergence of Transcript Isoforms in a Mammalian Model System](https://doi.org/10.1371/journal.pone.0137367). PLOS ONE 10 (9), e0137367. 

*	Latos P.A., Goncalves A., Oxley D., Mohammed H., Turro E. and Hemberger M. (2015) [Fgf and Esrrb integrate epigenetic and transcriptional networks that regulate self-renewal of Trophoblast Stem Cells](https://doi.org/10.1038/ncomms8776). Nature Communications 6. 

*	Ballester B.#, Medina-Rivera A., Schmidt D., Gonzalez-Porta M., Carlucci M., Chen X., Chessman K., Faure A.J., Funnell A., Goncalves A., Kutter C., Lukk M., Menon S., McLaren W.M., Stefflova K., Watt S., Weirauch M.T., Crossley M., Marioni J.C., Odom D.T., Flicek P. and Wilson M.D.# (2014) [Multi-species, multi-transcription factor binding highlights conserved control of tissue- specific biological pathways](https://doi.org/10.7554/eLife.02626). eLife 3, e02626. 

*	Goncalves A.#, Leigh-Brown S.#, Thybert D., Stefflova K., Turro E., Flicek P., Brazma A., Odom D.T. and Marioni J.C. (2012) [Compensatory cis-trans regulation dominates the evolution of mouse gene expression](https://doi.org/10.1101/gr.142281.112). Genome Research 22 (12), 2376-2384. 

*	Schmidt D.#, Schwalie P.C.#, Wilson M.D., Ballester B., Goncalves A., Kutter C., Brown G.D., Marshall A., Flicek P., and Odom D.T. (2012) [Waves of retrotransposon expansion remodel genome organization and CTCF binding in multiple mammalian lineages](https://doi.org/10.1016/j.cell.2011.11.058). Cell 148 (1), 335-348. 

*	Kutter C., Watt S., Stefflova K., Wilson M.D., Goncalves A., Ponting C.P., Odom D.T. and Marques A.C. (2012) [Rapid turnover of long noncoding RNAs and the evolution of gene expression](https://doi.org/10.1371/journal.pgen.1002841). PLOS genetics 8 (7), e1002841. 

*	Kutter C.#, Brown G.D.#, Goncalves A., Wilson M.D., Watt S., Brazma A., White R.J. and Odom D.T. (2011) [Pol III binding in six mammals shows conservation among amino acid isotypes despite divergence among tRNA genes](https://doi.org/10.1038/ng.906). Nature Genetics 43 (10), 948-955. 

*	Lukk M., Kapushesky M., Nikkil J., Parkinson H., Goncalves A., Huber W., Ukkonen E. and Brazma A. (2010) [A global map of human gene expression](https://doi.org/10.1038/nbt0410-322). Nature Biotechnology 28 (4), 322-324.

### Published independently by group members

* Baser A, Skabkin M, Kleber S, Dang Y, Gülcüler Balta G, Kalamakis G, Göpferich M, Carvajal Ibañez D, Schefzik R, Santos Lopez A, Llorens Bobadilla E, Schultz C, Fischer B and Martin-Villalba A. (2019) Onset of differentiation is post-transcriptionally controlled in adult neural stem cells, Nature 566:100-104. doi: 10.1038/s41586-019-0888-x.

* Zorea J, Prasad M, Cohen L, Li N, Schefzik R, Ghosh S, Rotblat B, Brors B, Elkabets M. (2018) IGF1R upregulation confers resistance to isoform-specific inhibitors of PI3K in PIK3CA-driven ovarian cancer. Cell Death Dis.

